core/src/main/java/org/p2p/core/token/Token.kt

```diff
@@ -269,3 +270,20 @@ fun List<Token.Active>.findByMintAddress(mintAddress: String): Token.Active? =
 @JvmName("findByNullableMintAddress")
 fun List<Token.Active>.findByMintAddress(mintAddress: String?): Token.Active? =
     mintAddress?.let(::findByMintAddress)
+
+fun List<Token.Active>.sortedWithPreferredStableCoins(): List<Token.Active> {
+    return sortedWith(
+        compareBy<Token.Active> {
+            when (it.tokenSymbol) {
+                USDC_SYMBOL -> 1
+                USDT_SYMBOL -> 2
+                else -> 3
+            }
+        }
+            .thenComparing(
+                compareByDescending {
```

Comment:
can be extracted to a val and passes here so it's more readable
same with compareBy<Token.Active> {